docs/examples/frameworks/jax/jax-basic_example.ipynb

```diff
@@ -0,0 +1,277 @@
+{
```

Comment:
<p>To get <strong>the</strong> iterator compatible with JAX we need to import <strong>it</strong>  from DALI JAX plugin.</p><p> In addition to the pipeline we can pass the output_map, reader_name and auto_reset <strong>parameters</strong> to the iterator.</p><p> **Here is a quick...**</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/4937/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>